numpy/_core/src/multiarray/array_method.c

```diff
@@ -143,7 +149,19 @@ npy_default_get_strided_loop(
     PyArray_Descr **descrs = context->descriptors;
     PyArrayMethodObject *meth = context->method;
     *flags = meth->flags & NPY_METH_RUNTIME_FLAGS;
-    *out_transferdata = NULL;
+    if (meth->aux_data == NULL) {
+        *out_transferdata = NULL;
+    }
+    else {
+        static_auxdata_struct *transferdata = (static_auxdata_struct *)malloc(sizeof(static_auxdata_struct));
```

Comment:
@seberg - I followed your example above, but defining a `.clone` might actually be nicer. But, really, I'm just messing around...

Reply:
@seberg - I now realize there is really no point creating this `transferdata` here since in the loop I can just access `context->method->aux_data`.  Is that what you had in mind? If so, I'll remove these changes in `array_method.*`...

Reply:
Right we shouldn't build an AuxData struct at all of course, and this second iteration is what what I had in mind.  Either `context->method->const_data` or `context->const_data`, and if all we do is copy it, the first seems just as well.

(Moving it to the `context` could be done to make it a true scratch space, but we can also just add scratch space to `context` for that.)

I think we should rename `aux_data`, though.  Since there already is the other auxdata and it has a per ufunc run lifetime!  I can see `static_data`, `const_data` or even using `context`/`ctx` again.

One thing I was mentioning (but let's defer that) is that I don't want the Method objects struct fully public, so on the public side it would make sense to move this high up, so we can decide to make parts public (I think it might be fully opaque right now).